content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -384,20 +384,20 @@ rules:
 ```
 
 Allow reading/writing Deployments (at the HTTP level: objects with `"deployments"`
-in the resource part of their URL) in both the `"extensions"` and `"apps"` API groups:
+in the resource part of their URL) in the `"apps"` API groups:
 
 ```yaml
 rules:
-- apiGroups: ["extensions", "apps"]
+- apiGroups: ["apps"]
   #
-  # at the HTTP level, the name of the resource for accessing Deployment
+  # at the HTTP level, the name of the resource for accessing Deployments
```

Comment:
nit:
```suggestion
  # at the HTTP level, the name of the resource for accessing Deployment
```
In the URL it's `/deployments/`; in text, it's “Deployments” or “Deployment objects”.

Reply:
Oh no, it's a typo when I manually revert the change. :cry:

How could I miss that, thanks for pointing it out! :bowing_man: